source/adios2/toolkit/sst/cp/ffs_marshal.c

```diff
@@ -1552,14 +1552,7 @@ extern void SstFFSWriterEndStep(SstStream Stream, size_t Timestep)
 
     /* free all those copied dimensions, etc */
     MBase = Stream->M;
-    size_t *tmp = MBase->BitField;
-    /*
-     * BitField value is saved away from FMfree_var_rec_elements() so that it
-     * isn't unnecessarily free'd.
-     */
-    MBase->BitField = NULL;
     FMfree_var_rec_elements(Info->MetaFormat, Stream->M);
-    MBase->BitField = tmp;
     FMfree_var_rec_elements(Info->DataFormat, Stream->D);
     memset(Stream->M, 0, Stream->MetadataSize);
```

Comment:
@eisenhauer The problem is that the pointer is being cleared here. This happened in 1d64c264.

Reply:
Ah yes, it's cleared in the memset.  So, I think the preferred fix is to move the MBase->BItField = tmp; assignment to after the memset(Stream->M...).

Reply:
Understood. I figured you had a good reason for the `memset()`, so I took the conservative approach of `free()`ing it before the `memset()`. Will fix.